R/missingDataImputation.R

```diff
@@ -19,19 +19,20 @@
 
 #' Multiply impute missing data with MICE
 #' @export
-MissingDataImputation <- function(jaspResults, dataset, options) {
 
-  # saveRDS(dataset, "~/software/jasp/modules/imputation/data/dataset.rds")
-  # browser()
+# TODO: own logging
+
+MissingDataImputation <- function(jaspResults, dataset, options) {
 
   # Set title
   jaspResults$title <- "Multiple Imputation with MICE"
 
   # Init options: add variables to options to be used in the remainder of the analysis
   options <- .processImputationOptions(options)
 
-  # ready <- length(options$variables) > 0
-  if (.readyForMi(options)) {
```

Comment:
We need this ready check. Otherwise, mice tries to run without any variables to impute.

Reply:
Solved in [85854c6](https://github.com/jasp-missing-data/jaspMissingData/pull/18/commits/85854c6497f3fba4d8cb23ef7db422ff54da2882)